CLibExtensions/include/CLib/Vector.h

```diff
@@ -13,7 +13,7 @@ typedef struct tagVector Vector;
 
 // Creates a new vector object, allocating an initial size.
 CLIBRARY_API bool VectorCreate(unsigned int dataTypeSize, unsigned int initialCapacity, unsigned int maxCapacity,
-	EqualsFunction* equalImpl, Vector** vect);
+							   EqualsFunction* equalsImpl, CompareToFunction* compareImpl, Vector** vect);
```

Comment:
any particular reason why the spacing went awol here?

Reply:
You right, the spacing went nuts, lol.